wagtail/wagtailadmin/rich_text.py

```diff
@@ -31,6 +32,15 @@ def value_from_datadict(self, data, files, name):
             return None
         return DbWhitelister.clean(original_value)
 
+    @property
+    def media(self):
+        return Media(js=[
+            static('wagtailadmin/js/vendor/hallo.js'),
+            static('wagtailadmin/js/hallo-plugins/hallo-wagtaillink.js'),
+            static('wagtailadmin/js/hallo-plugins/hallo-hr.js'),
+            static('wagtailadmin/js/hallo-plugins/hallo-requireparagraphs.js'),
+        ])
+
```

Comment:
You don't need to wrap these in `static(...)` - Django will do this automatically for any non-absolute paths (see https://docs.djangoproject.com/en/1.9/topics/forms/media/#paths-in-asset-definitions, https://github.com/django/django/blob/86573861a95e5a47dc7ff906443117d75b73dca1/django/forms/widgets.py#L91)